tools/pythonpkg/src/numpy/array_wrapper.cpp

```diff
@@ -285,6 +285,22 @@ struct ListConvert {
 	}
 };
 
+struct ArrayConvert {
+	static py::tuple ConvertValue(Vector &input, idx_t chunk_offset, const ClientProperties &client_properties) {
+		auto val = input.GetValue(chunk_offset);
```

Comment:
In light of https://github.com/duckdb/duckdb/pull/10826 reworking `ListConvert` - this should ideally also just call/be unified with `ListConvert`, but perhaps @Tishj can pick that up in his rework